src/Avalonia.Base/Platform/StandardRuntimePlatform.cs

```diff
@@ -14,42 +14,10 @@ public IDisposable StartSystemTimer(TimeSpan interval, Action tick)
 
         public IUnmanagedBlob AllocBlob(int size) => new UnmanagedBlob(size);
         
-        private static readonly Lazy<RuntimePlatformInfo> Info = new(() =>
+        private static readonly Lazy<RuntimePlatformInfo> Info = new(() => new RuntimePlatformInfo
         {
-            OperatingSystemType os;
-
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-                os = OperatingSystemType.OSX;
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
-                os = OperatingSystemType.Linux;
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                os = OperatingSystemType.WinNT;
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("Android")))
-                os = OperatingSystemType.Android;
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("iOS")))
-                os = OperatingSystemType.iOS;
-            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("Browser")))
-                os = OperatingSystemType.Browser;
-            else
-                throw new Exception("Unknown OS platform " + RuntimeInformation.OSDescription);
-
-            // Source: https://github.com/dotnet/runtime/blob/main/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs
-            var isCoreClr = Environment.Version.Major >= 5 || RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase);
-            var isMonoRuntime = Type.GetType("Mono.Runtime") != null;
-            var isFramework = !isCoreClr && RuntimeInformation.FrameworkDescription.StartsWith(".NET Framework", StringComparison.OrdinalIgnoreCase);
-            
-            return new RuntimePlatformInfo
-            {
-                IsCoreClr = isCoreClr,
-                IsDotNetFramework = isFramework,
-                IsMono = isMonoRuntime,
-                
-                IsDesktop = os is OperatingSystemType.Linux or OperatingSystemType.OSX or OperatingSystemType.WinNT,
-                IsMobile = os is OperatingSystemType.Android or OperatingSystemType.iOS,
-                IsUnix = os is OperatingSystemType.Linux or OperatingSystemType.OSX or OperatingSystemType.Android,
-                IsBrowser = os == OperatingSystemType.Browser,
-                OperatingSystem = os,
-            };
+            IsDesktop = OperatingSystemEx.IsWindows() || OperatingSystemEx.IsMacOS() || OperatingSystemEx.IsLinux(),
+            IsMobile = OperatingSystemEx.IsAndroid() || OperatingSystemEx.IsAndroid()
```

Comment:
I think wrapping them in a `Lazy<>` is a bad decision.

Reply:
Yes, makes sense to remove Lazy now